Project List/scripts.js

```diff
@@ -0,0 +1,15 @@
+const hide_buttons = document.getElementsByClassName("btn");
+Array.from(hide_buttons).forEach((button) => {
+    button.addEventListener('click',() => {
+        const element = button.parentNode;
+        element.style.display = 'none';
+    } )
+})
+
+const unhid = document.getElementById("unhide");
```

Comment:
use complete variable name `unhid => unhideButtons`

Reply:
solved in the latest merge